folly/portability/OpenSSL.h

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <openssl/base.h>
```

Comment:
This is causing build failures. Looks like this is only in BoringSSL. Needs a guard.